middlewares/crossOriginIsolation.js

```diff
@@ -0,0 +1,13 @@
+// For the given question, sets Cross-Origin-Opener-Policy to 'same-origin' and
+// Cross-Origin-Embedder-Policy: 'require-corp' if cross_origin_isolated is set to true
+//
+// Beneficial for questions that rely on SharedArrayBuffers.
```

Comment:
We generally prefer `/** */` comments so that Intellisense and other IDE tools can pick up on it. Also editing to focus more on the "why" than the "what" since this is such a simple function.

```suggestion
/**
 * Allows questions to opt in to cross-origin isolation. This is useful for
 * questions that rely on advanced features like `SharedArrayBuffer`.
 *
 * @see https://developer.mozilla.org/en-US/docs/Web/API/crossOriginIsolated
 */
```